validate.sh

```diff
@@ -30,3 +30,9 @@ else
 fi
 
 go test $(go list ./... | grep -v /vendor/)
+
+if $VET; then
```

Comment:
Before this check, `VET` is not being set above anywhere. Should we initialize in a Prebid Server fashion?
```
 1   #!/bin/bash
 2  
 3   set -e
 4  
 5   AUTOFMT=true
   + VET=true
   +
 6  
 7   while true; do
 8     case "$1" in
 9        --nofmt ) AUTOFMT=false; shift ;;
   +      --novet ) VET=false; shift ;;
10        * ) break ;;
11     esac
12   done
13  
14   die() { echo -e "$@" 1>&2 ; exit 1;  }
15   *-- 18 lines: check there are no formatting issues-------------------------
33  
34   if $VET; then
35     COMMAND="go vet"
36     echo "Running: $COMMAND"
37     `$COMMAND`
38   fi
```